### ISSUE TITLE
Inconsistent CSS and fonts through CDN

### ISSUE THREAD

**Issue by oyving** [denali-design/denali-css#103]
With the latest release of Denali we are seeing inconsistent versions of the CSS and fonts being delivered by jsdelivr. This is causing issues with the wrong icons appearing on the website. It appears that some clients are getting the old CSS but the new font.

The clients that are seeing the correct icons have a font request:
> https://cdn.jsdelivr.net/npm/denali-icon-font/dist/denali-icon-font.woff?1596055975302

The clients that are getting the wrong icons have font request:
> https://cdn.jsdelivr.net/npm/denali-icon-font/dist/denali-icon-font.woff?1586285317730

Since the timestamp in the request isn't respected or used by the CDN (perhaps beyond cache control) we are getting new fonts with the old CSS.

**Comment by frodelu** [comment]
Example of inconsistency include previous version of CSS mapping:
```
.d-tools-solid:before {
 content: '\ee10';
}
.d-tools:before {
 content: '\ee11';
}
```

vs. new version:

```
.d-thumbs-down-solid:before {
 content: '\ee10';
}

.d-thumbs-down:before {
 content: '\ee11';
}

(...)

.d-tools-solid:before {
 content: '\ee2e';
}
.d-tools:before {
 content: '\ee2f';
}
```

As a result, the `tools` icon will show a `thumbs-down` icon when the old CSS is combined with the new font file. 

Understanding that the actual CSS-font-mapping is auto-generated, is there any way to minimize moving icons around in the font file when new icons are added? Having the numbering follow the time of addition rather than alphabetically from the directory?

**Comment by chasturansky** [comment]
Hi @frodelu and @oyving 

Thank you for bringing this up. I believe this is because of caching in the browser. Since this font is auto-generated it is more difficult to change that process. As a solution, I would recommend using the CDN and picking a specific version.

**Comment by chasturansky** [comment]
Hi @frodelu and @oyving

Thank you for bringing this up. I believe this is because of caching in the browser. Since this font is auto-generated it is more difficult to change that process, but we will look into adding new icons by the time of addition. 

As a solution in the meantime, I would recommend using the CDN and picking a specific version like this.
`https://cdn.jsdelivr.net/npm/denali-icon-font@1.5.0/dist/denali-icon-font.css` just specific `@1.5.0`

**Comment by frodelu** [comment]
Hi @chasturansky 

We are quite certain this was not a browser cache issue as we could replicate different behavior using `curl` from two different laptops:
```
$ curl https://cdn.jsdelivr.net/npm/denali-icon-font/dist/denali-icon-font.css -s |md5
e243e4ef45eb7337b0cec291cc75e4fe
```

vs.

```
$ curl https://cdn.jsdelivr.net/npm/denali-icon-font/dist/denali-icon-font.css -s |md5
36235b077d55124bf830400e38c9f30c
```

The two above commands were run at the same time on two different computers (connected through different network paths to the CDN) and the former consistently gave the old CSS while the latter consistently gave the new CSS. That said - such CDN propagation delays are probably expected through CDNs so I believe the Denali Icons release process need to be adapted to account for this.

Two observations:
1. It seems the matching of correct font file version to CSS is managed by using the release timestamp in the query part of the URL as a "cache buster", but while many caches would ignore the query part, this would potentially bust the browser cache, but not necessarily the CDNs. Could one force the timestamp as part of the font filename to ensure the correct mapping?
1. The problem could be minimized/mitigated, but limiting re-ordering of icons in the font file between releases. If new icons were added "to the end" of font in terms of character codes, old CSS would work with new font file and new CSS would mostly work with old font file except for new icons. This seems to be the strategy of e.g. Font Awesome where they have a static mapping from icon names to character code in the font.

**Comment by chasturansky** [comment]
So I did some research on this and have some good news. We are currently using [gulp-iconfont](https://www.npmjs.com/package/gulp-iconfont) to build our icons. There is an option to prepend unicode codepoints to icon file names. This will keep codepoints consistent across each build. I'm going to test and let you know when this is complete.

**Comment by chasturansky** [comment]
This is now fixed in v1.7
